examples/fluids/README.md

```diff
@@ -7,390 +7,499 @@ The state variables are mass density, momentum density, and energy density.
 
 The main Navier-Stokes solver for libCEED is defined in [`navierstokes.c`](navierstokes.c) with different problem definitions according to the application of interest.
 
-Build by using
+Build by using:
 
 `make`
 
-and run with
+and run with:
 
-`./navierstokes`
+```
+./navierstokes -ceed [ceed] -problem [problem type] -degree [degree]
+```
 
-Available runtime options for all problem cases are:
+## Runtime options
 
-|  Option                         | Meaning                                                                             | Default                 |
-| :-------------------------------| :-----------------------------------------------------------------------------------| :-----------------------|
-| `-ceed`                         | CEED resource specifier                                                             | `/cpu/self/opt/blocked` |
-| `-test`                         | Run in test mode                                                                    | `false`                 |
-| `-compare_final_state_atol`     | Test absolute tolerance                                                             | `1E-11`                 |
-| `-compare_final_state_filename` | Test filename                                                                       |                         |
-| `-problem`                      | Problem to solve (`advection`, `advection2d`, `density_current`, or `euler_vortex`) | `density_current`       |
-| `-implicit`                     | Use implicit time integartor formulation                                            | `false` (`explicit`)    |
-| `-viz_refine`                   | Use regular refinement for visualization                                            | `0`                     |
-| `-degree`                       | Polynomial degree of tensor product basis (must be >= 1)                            | `1`                     |
-| `-q_extra`                      | Number of extra quadrature points                                                   | `2`                     |
-| `-output_freq`                  | Frequency of output, in number of steps                                             | `10`                    |
-| `-continue`                     | Continue from previous solution                                                     | `0`                     |
-| `-output_dir`                   | Output directory                                                                    | `.`                     |
+% inclusion-fluids-marker
 
-### Advection
+The Navier-Stokes mini-app is controlled via command-line options, the following of which are common among all problem cases.
```

Comment:
```suggestion
The Navier-Stokes mini-app is controlled via command-line options.
The following options are common among all problem cases:
```

"problem cases" feels a little awkward to me?

Reply:
Haha, right! `problems` would be enough!